airflow_config/Dockerfile

```diff
@@ -26,22 +26,28 @@ RUN apt update -y && apt-get install -y --no-install-recommends \
         ldap-utils \
         libffi6 \
         libffi-dev \
+        libldap2-dev \
```

Comment:
Are we installing ldap libraries "just in case" we want to use ldap with airflow too?